content/en/observability_pipelines/setup/datadog.md

```diff
@@ -347,6 +356,33 @@ EOT
 }
 ```
 {{% /tab %}}
+{{% tab "CloudFormation" %}}
+To install the Worker in your AWS Account, use the CloudFormation template to create a Stack:
+
+  1. Download [the CloudFormation template][1] for the Worker.
+
+  2. In the **CloudFormation console**, click **Create stack**, and select the **With new resources (standard)** option.
+
+  3. Make sure that the **Template is ready** option is selected, and select **Upload a template file**. Click **Choose file** and add the CloudFormation template file you downloaded earlier. Click **Next**.
+
+  4. Enter a name for the stack in **Specify stack details**.
+
+  5. Fill in the parameters for the CloudFormation template. A few require special attention:
+
+  * For **APIKey** and **PipelineID**, provide the key and ID that you gathered earlier in the Prerequisites section.
+    
+  * For the **VPCID** and **SubnetIDs**, provide the subnets and VPC you chose earlier.
+
+  * All other parameters are set to reasonable defaults for a Worker deployment but you can adjust them for your use case as needed.
```

Comment:
```suggestion
      * For **APIKey** and **PipelineID**, provide the key and ID that you gathered earlier in the Prerequisites section.
    
      * For the **VPCID** and **SubnetIDs**, provide the subnets and VPC you chose earlier.

      * All other parameters are set to reasonable defaults for a Worker deployment but you can adjust them for your use case as needed.
```

Reply:
Sadly this won't work, I tried it before and it just created a code block with the bullet points inside. :(